src/node/interfaces.cpp

```diff
@@ -155,6 +158,66 @@ class GOVImpl : public GOV
         error = "Governance manager not available";
         return false;
     }
+    GovernanceInfo getGovernanceInfo() override
+    {
+        GovernanceInfo info;
+        if (context().chainman) {
+            const CBlockIndex* tip = WITH_LOCK(::cs_main, return context().chainman->ActiveChain().Tip());
+            int last = 0, next = 0;
+            const int height = tip ? tip->nHeight : 0;
+            CSuperblock::GetNearestSuperblocksHeights(height, last, next);
+            info.lastsuperblock = last;
+            info.nextsuperblock = next;
+        }
+        info.proposalfee = GOVERNANCE_PROPOSAL_FEE_TX;
+        info.superblockcycle = Params().GetConsensus().nSuperblockCycle;
```

Comment:
nit: refactor: replace consequent calls of Params().GetConsensus() to local variable; same for context()